lib/MTT/Reporter/HTMLReport.pm

```diff
@@ -238,11 +242,24 @@ sub _summary_report
 
     ";
 
+    my $ignore_min_succ = 0;
+    my $extra_info = "";
+    if (not $total_tests) {
+        $extra_info = "<h2>Extra info</h2><span style='color:red'>No tests were executed.</span>";
+        if (MTT::Util::time_to_terminate()) {
+            $extra_info .= "\n<span>Terminate file found.</span>";
```

Comment:
Do you want to put this extra info in (that the terminate file was found) regardless of whether any tests were run or not?

Reply:
No, I use it as a tip to help understand why no there were no tests.

Reply:
What I mean is: the Terminate message is relevant regardless of whether any tests were run or not.  E.g., I periodically `touch terminate-file` to drain my SLURM queues of MTT jobs in an orderly fashion -- the currently running jobs will stop in the middle (and therefore have `$total_tests>0`.  All jobs after that will have `$total_tests==0` (because they'll notice the terminate file immediately and not actually run any tests).  The point is that both cases can (and do) happen -- so it is probably worthwhile to pull the "Terminate file found" message out of the check for how many `$total_tests` have run.

For debugging purposes, you'll still get your Extra Info from `No tests were run`, but it's *also* useful to know that the overall MTT job completed because of a termination file.

Make sense?

Reply:
Got you. Make sense, will update.